release/src/main/scripts/run_rc_validation.sh

```diff
@@ -520,42 +454,36 @@ if [[ $confirmation = "y" ]]; then
       sleep 25m
       echo "* How to verify results:"
       echo "* 1. Check whether there is any error messages in the task running terminal."
-      echo "* 2. Goto your BigQuery console and check whether your ${MOBILE_GAME_DATASET} has game_stats_teams and game_stats_sessions table."
+      echo "* 2. Goto your BigQuery console and check whether your ${GAMESTATS_DIRECT_DATASET} has game_stats_teams and game_stats_sessions table."
       echo "* 3. Check whether game_stats_teams has data, retrieving BigQuery data as below: "
-      bq head -n 10 ${MOBILE_GAME_DATASET}.game_stats_teams
+      bq head -n 10 ${GAMESTATS_DIRECT_DATASET}.game_stats_teams
       echo "* 4. Check whether game_stats_sessions has data, retrieving BigQuery data as below:"
-      bq head -n 10 ${MOBILE_GAME_DATASET}.game_stats_sessions
+      bq head -n 10 ${GAMESTATS_DIRECT_DATASET}.game_stats_sessions
       echo "***************************************************************"
-
-      echo "If you have verified all items listed above, please terminate the python job."
-      echo "[Confirmation Required] Please confirm whether you have stopped this job: [y|N]"
-      read confirmation
-      if [[ $confirmation != "y" ]]; then
-        echo "Current job must be terminated in order to proceed into next test."
-        clean_up
-        exit
-      fi
+    else
+      echo "* Skip Python GameStats with DirectRunner"
     fi
 
     echo "-------------------Starting GameStats with DataflowRunner--------------------"
-    echo "[Confirmation Required] Do you want to proceed? [y|N]"
-    read confirmation
-    if [[ $confirmation = "y" ]]; then
-      bq rm -rf --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
-      bq mk --project=${USER_GCP_PROJECT} ${MOBILE_GAME_DATASET}
+    if [[ "$python_gamestats_dataflow" = true ]]; then
+      GAMESTATS_DF_DATASET=${USER}_python_validations_$(date +%m%d)_$RANDOM
+      bq mk --project=${USER_GCP_PROJECT} ${GAMESTATS_DF_DATASET}
+      echo "export GAMESTATS_DF_DATASET=${GAMESTATS_DF_DATASET}" >> ~/.bashrc
+
       echo "This is a streaming job. This task will be launched in a separate terminal."
       echo "Streaming job is running with fixed_window_duration=${FIXED_WINDOW_DURATION}"
       gnome-terminal -x sh -c \
```

Comment:
this doesn't work if I'm ssh-ing into my machine

Reply:
I got same thing. I'll sync with Yifan tomorrow to see if we can solve painpoints coming from gnome-terminal.